test/account.c

```diff
@@ -130,6 +130,9 @@ int test_account_uri_complete(void)
 	err = account_alloc(&acc, "\"A\" <sip:A@proxy.com>");
 	TEST_ERR(err);
 
+	err = account_set_regint(acc, 3600);
+	TEST_ERR(err);
+
```

Comment:
I think the regint should be zero for testcode. Can you run the testcode and make sure
the program does not send any packets to proxy.com ?